WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Method/SimplePaymentsMethodsViewModel.swift

```diff
@@ -103,6 +107,7 @@ final class SimplePaymentsMethodsViewModel: ObservableObject {
          formattedTotal: String,
          presentNoticeSubject: PassthroughSubject<SimplePaymentsNotice, Never> = PassthroughSubject(),
          cppStoreStateObserver: CardPresentPaymentsOnboardingUseCaseProtocol = CardPresentPaymentsOnboardingUseCase(),
+         cardPresentPaymentsOnboardingPresenter: CardPresentPaymentsOnboardingPresenting = CardPresentPaymentsOnboardingPresenter(),
```

Comment:
Extremely personal preference, but I tend to cringe when I see so many dependencies injected via the constructor: the list tends to get longer and longer with time.

Would it make sense to wrap these dependencies in a single struct and inject said struct?

Reply:
Sure, I'll do that 😊

It does also have the potential benefit of making it easier to update the tests when you add new ones.